inst/recorder/app.R

```diff
@@ -390,8 +390,23 @@ shinyApp(
     })
     outputOptions(output, "recorder_js", suspendWhenHidden = FALSE)
 
+    # echo console output from the driver object (in real-time)
+    if (!identical(debug, "none")) {
+      nConsoleLines <- 0
+      observe({
+        invalidateLater(500)
+        logs <- app$getDebugLog(debug)
+        n <- nrow(logs)
+        if (n > nConsoleLines) {
+          newLines <- seq.int(nConsoleLines + 1, n)
+          print(logs[newLines, ])
```

Comment:
Maybe add an argument to the `print` method to print a short form of log output, which would look like this:

```
C> Listening on port 1233
C>
B> Some browser message here
```

And also color the two differently.